mappings/net/minecraft/world/storage/SerializingRegionBasedStorage.mapping

```diff
@@ -34,8 +37,13 @@ CLASS net/minecraft/class_4180 net/minecraft/world/storage/SerializingRegionBase
 		ARG 1 pos
 		ARG 3 data
 	METHOD method_20369 getDataVersion (Lcom/mojang/serialization/Dynamic;)I
+		ARG 0 dynamic
 	METHOD method_20370 save (Lnet/minecraft/class_1923;)V
+		ARG 1 chunkPos
 	METHOD method_20436 saveChunk (Lnet/minecraft/class_1923;)V
 		ARG 1 pos
 	METHOD method_20621 loadNbt (Lnet/minecraft/class_1923;)Lnet/minecraft/class_2487;
 		ARG 1 pos
+	METHOD method_33637 chunkPosAsLong (Lnet/minecraft/class_1923;I)J
```

Comment:
I think `chunkSectionPosAsLong` is more accurate. It converts a pair of a `chunkPos` and `y` (which is essentially the same as `ChunkSectionPos`) to a long.
```suggestion
	METHOD method_33637 chunkSectionPosAsLong (Lnet/minecraft/class_1923;I)J
```